desktop-app/app/containers/Root.js

```diff
@@ -57,19 +69,67 @@ const getApp = history => {
 
 export default class Root extends Component<Props> {
   componentDidMount() {
-    // const {store} = this.props;
+    this.registerAllShortcuts();
+  }
+
+  registerAllShortcuts = () => {
+    const {store} = this.props;
+
+    registerShortcut({id: 'ZoomIn', title: 'Zoom In', accelerators: ['mod+plus', 'mod+shift+=']}, () => {
```

Comment:
@jjavierdguezas Just curious, which key does the `mod` here refer too?

Reply:
this is the Mousetrap sintax, it is the equivalent of `CommandOrCtrl` you can get more info [here](https://craig.is/killing/mice)

Reply:
That is great, thanks for the info JJ.